### DISCUSSION TITLE
Uno.Sdk 5.3.0-dev.1916 uses Uno 5.4.0 preview references

### DISCUSSION THREAD

**Discussion by inforithmics** [unoplatform/uno#17589] [General]
### Current behavior

Uno.Sdk 5.3.0-dev.1916 uses Uno 5.4.0 preview references

### Expected behavior

Uno.Sdk 5.3.0-dev.1916 uses only 5.3.0 uno references

### How to reproduce it (as minimally and precisely as possible)

Create new uno app
set global.json the uno.sdk to 

"Uno.Sdk": "5.3.0-dev.1873"

### Workaround

using 5.3.0-dev.1873 is the highest version that still uses uno 5.3 preview references
"Uno.Sdk": "5.3.0-dev.1916" uno 5.4 preview references are used.

### Works on UWP/WinUI

None

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

Uno.Sdk 5.3.0-dev.1916

### Affected platforms

Build tasks

### IDE

Visual Studio 2022

### IDE version

17.10

### Relevant plugins

Resharper

### Anything else we need to know?

Maybe this is intended behavior but this means a final uno 5.3 sdk will use 5.4 uno nugets which seems confusing

**Comment by jeromelaban** [comment]
Thanks for the report. This is expected and since it's published packages, we can't change it. You can use 5.4-dev sdk instead.

What are you trying to achieve by using these dev packages?

**Reply by inforithmics** [reply]
I was trying to solve https://github.com/unoplatform/uno/issues/17516 using a preview version of uno fixed it. This is used in this pull request https://github.com/Mapsui/Mapsui/pull/2603. This pull request will go into Mapsui when all components are offical versions. So I wanted to point out this uno.sdk behavior so that it isn't present in the Release Version of the uno.Sdk.

**Reply by jeromelaban** [reply]
I see! So at this time, Mapsui does not need to build with a recent version of the uno, but the app using it should be built using a 5.3 build (not yet released) or later.

**Reply by inforithmics** [reply]
Yes that‘s correct
